alarms/Telegram/telegram_alarm.py

```diff
@@ -64,10 +70,28 @@ def set_alert(self, settings, default):
 		alert['title'] = settings.get('title', default['title'])
 		alert['body'] = settings.get('body', default['body'])
 		alert['location'] = parse_boolean(settings.get('location', default['location']))
+		alert['sticker_id'] = settings.get('send_stickers', default['sticker_id'])
 		return alert
  		
 	#Send Alert to Telegram
  	def send_alert(self, alert, info):
```

Comment:
The idea is that the sticker will be optional - so if no sticker_id is included, it won't add it to the args. By defaulting it to None, we can check if it is None (with the comment below) and only add it if its not. `self.sticker` is only going to be a boolean to enable/disable the sticker sending (for people who don't want stickers).